apps/admin-x-activitypub/src/MainContent.tsx

```
@@ -2,23 +2,43 @@ import Inbox from '@views/Inbox';
+const Content: React.FC<ContentProps> = ({route}) => {
+    switch (route) {
     case 'search':
         return <Search />;
     case 'notifications':
         return <Notifications />;
     case 'profile':
         return <Profile />;
     default:
-        const layout = (mainRoute === 'inbox' || mainRoute === '') ? 'inbox' : 'feed';
-
+        const layout = (route === 'inbox' || route === '') ? 'inbox' : 'feed';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)